src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/rules-list/rules-list.component.html

```diff
@@ -20,3 +20,18 @@
                       [hideKeys]="hideKeys">
   </cd-table-key-value>
 </cd-table>
+
+<ng-template #severityTpl
+             let-row="row">
+  <span class="badge"
+        [ngClass]="row | pipeFunction:getSeverityClass">
+    {{ row.labels.severity }}
+  </span>
+</ng-template>
```

Comment:
This is not required. The cd-datatable already contain the `CellTransformation` field that allows one to convert [cell text into badges](https://github.com/ceph/ceph/pull/46836/files#diff-67c98e358295ce647a9754e11467d30e57e9a9478f34814d4fd7a554c2e4bd63R73-R79). Rather than implementing ad-hoc decoration for each specific table, I'd suggest that we invest in learning, reusing and (if needed) implementing decoration that can be reused.